Cargo.lock

```diff
@@ -254,6 +260,11 @@ dependencies = [
  "winapi 0.2.8 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
```

Comment:
Why does this ad d the shell-escape package?

Reply:
to be honest, I don't know. I just run `cargo update -p lazy_static`